Add tabs instead of messing with navigation (#20)
vignetteapp/docs: docs/install/install-methods.md

---
id: install-methods
title: Installation
sidebar_position: 2
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Installation Methods

Vignette provides a lot of official ways for you to get the software. We have listed what we support so far below, however, if you don't see your target platform, please [raise an issue](https://github.com/vignetteapp/vignette/issues/new) for your platform and see if we can support it.

<Tabs groupId="operating-systems">
  <TabItem value="microsoft-store" label="Microsoft Store (Windows)" default>
  
  Vignette is Coming Soon to the Microsoft Store! 
  
  
  Until then, please consinder installing it via our Portable Installation Guide
  </TabItem>
  <TabItem value="steam" label="Steam">
  Vignette is Coming Soon to Steam!
  

>   Until then, please consinder installing it via our Portable Installation Guide or AUR Installation Guide.

Reviewer: Maybe remove this new line to match with Windows Store tab to be consistent? or vice versa
![image](https://user-images.githubusercontent.com/34704796/135729904-a1b6936d-3d9f-492c-99fa-2cc366887628.png)
![image](https://user-images.githubusercontent.com/34704796/135729909-aeb54564-7935-4192-8ab2-7a64707c3769.png)

Author: Wanted to add it to the Microsoft Store one too,  but it didn't actually put the newline there.

Reviewer: Hmm, maybe try adding double spaces at the end.
Ex: `Vignette is Coming Soon to the Microsoft Store!  Until then, ...`

  </TabItem>
  <TabItem value="aur" label="AUR (Linux)">

  If you prefer, to install Vignette on Linux we provide the official [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/) AUR package. This guide will be using `yay`.

## Installation process

In order to install Vignette using yay, run the following command:

`yay vignette-git`

If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.

![Image of yay listing package, with aur/vignette-git on position 1](/img/install/yay-terminal-window.webp)

:::note 

When prompted for your password for sudo, enter it to proceed with the installation.

> :::

Reviewer: I'm not sure if this is necessary since it should be already clear (and you're speaking to Linux users lol)

Author: Was already there... I kept the page as it was.

Reviewer: I think it could be removed but it's also fine to keep it (maybe change the wording to something like suggestion)

```suggestion
:::note 

When prompted, enter your password so it could proceed with the installation.

:::
```

Author: I think that's out of scope for this PR, but we can do that on Ayyrik's branch directly

Reviewer: Sounds fine to me
